### PR TITLE
add focalmap users

### PR DESCRIPTION
**Pull request by johrstrom**
This adds focalmap users to the container. @treydock if you want to mount SSSD into the containers, I'm happy to explore that option too.

### CODE REVIEW

File: user-setup.sh [hunk 1]
```diff
@@ -10,21 +10,24 @@ set -xe
...
 groupadd -g 6493 PAS1694
 groupadd -g 6602 PZS1008
 groupadd -g 6603 accesscovidsim
 groupadd -g 6604 accesscovidsimdev
+groupadd -g 7175 PDE0001
 
 useradd -g PAS1694 -u 33024 covidsim
 useradd -g PAS1694 -u 33031 covidsimdev
+useradd -g PDE0001 -u 36970 focalmap
+useradd -g PDE0001 -u 36971 focalmapdev
```
Comment (treydock) [inline user-setup.sh]: I would recommend making these look like how they are outside container:

```suggestion
useradd -g PZS0523 -G PDE0001 -u 36971 focalmapdev
```

File: user-setup.sh [hunk 2]
```diff
@@ -10,21 +10,24 @@ set -xe
...
 
 groupadd -g 6697 PAS1788
-useradd -g PAS1788 -u 33929 covid
-useradd -g PAS1788 -u 33930 coviddev
-
 groupadd -g 6493 PAS1694
 groupadd -g 6602 PZS1008
 groupadd -g 6603 accesscovidsim
 groupadd -g 6604 accesscovidsimdev
+groupadd -g 7175 PDE0001
```
Comment (treydock) [inline user-setup.sh:20]: Also need primary group:

```suggestion
groupadd -g 7175 PDE0001
groupadd -g 2925 PZS0523
```

File: user-setup.sh [hunk 3]
```diff
@@ -10,21 +10,24 @@ set -xe
...
 
 useradd -g PAS1694 -u 33024 covidsim
 useradd -g PAS1694 -u 33031 covidsimdev
+useradd -g PDE0001 -u 36970 focalmap
```
Comment (treydock) [inline user-setup.sh]: Same as other, good to make look like outside container:
```suggestion
useradd -g PZS0523 -G PDE0001 -u 36970 focalmap
```

**Comment by treydock** [comment]
Making the primary group and secondary group look like outside container might be necessary but it might given the app code and home directory are owned by primary group PZS0523 and the secondary group is used for data access. I plan to make the container run as secondary group.

**Comment by treydock** [comment]
Also I think making this work with SSSD is a heavier lift that would require a lot more testing, so I think this simple change to get the app deployed is good and then we can circle back and figure out SSSD necessary changes.

File: user-setup.sh (hunk 2 above)
Comment (johrstrom) [reply user-setup.sh:20]: done.

File: user-setup.sh (hunk 3 above)
Comment (johrstrom) [reply user-setup.sh]: done.

File: user-setup.sh (hunk 1 above)
Comment (johrstrom) [reply user-setup.sh]: done.

**Review by treydock — APPROVED** [review]

**Comment by johrstrom** [comment]
Can u take a second look? I regrouped the covid users and accidentally deleted them. It should be good now, but you can't rule out a copy paste error.

**Review by treydock — APPROVED** [review]

**Comment by treydock** [comment]
Looks fine to me.

**Pull request merged**